WalletWasabi/WabiSabi/ProtocolConstants.cs

```diff
@@ -14,8 +14,8 @@ public static class ProtocolConstants
 
 	public const string RoundAllowedInputAmountsStrobeLabel = "allowed-input-amounts";
 	public const string RoundAllowedOutputAmountsStrobeLabel = "allowed-output-amounts";
-	public const string RoundAllowedInputTypesStrobeLabel = "allowed-input-types";
-	public const string RoundAllowedOutputTypesStrobeLabel = "allowed-output-types";
+	public const string RoundAllowedInputScriptTypesStrobeLabel = "allowed-input-script-types";
```

Comment:
This introduces a breaking change in the protocol because the clients will compute the round hash using the previous values. I would suggest to revert this and we can consider introduce it again in a major version.

Reply:
> clients will compute the round hash using the previous values

Do they? I didn't find any place where clients compute the round hash.

Reply:
f%$#"/%&!

Reply:
I seems i made a mistake. It can be fixed.
Your change doesn't break anything.
@molnard